Targets/UserInterface/Sources/Utils/Extensions/View+BottomSheet.swift

```diff
@@ -9,10 +9,10 @@
 import SwiftUI
 
 extension View {
-    func ticlmoaBottomSheet(isPresented: Binding<Bool>, @ViewBuilder content: @escaping () -> (some View)) -> some View {
+    func ticlmoaBottomSheet(isPresented: Binding<Bool>, withHandleBar: Bool = false, @ViewBuilder content: @escaping () -> (some View)) -> some View {
         let keyWindow = UIApplication.shared.connectedScenes.compactMap { ($0 as? UIWindowScene)?.keyWindow }.first!
```

Comment:
```suggestion
        let keyWindow = UIApplication.shared.connectedScenes.compactMap { ($0 as? UIWindowScene)?.keyWindow }.first
```

ticlmoaBottomSheet 메소드 호출하는 View에서 강제 추출로 인해 preViews 실패가 되는데 강제 추출 제거해 주실 수 있으신가요?

Reply:
#102 에서 수정하겠습니다. 감사합니다.